src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -169,6 +170,31 @@ public FeedbackResponseCommentAttributes updateFeedbackResponseComment(
             throw new InvalidParametersException(newAttributes.getInvalidityInfo());
         }
 
+        // update only if change
+        boolean hasSameAttributes =
```

Comment:
The name should be `isSameAttribute` ?

Reply:
hasSameAttributes = (The entity before and after the update) has same attributes.